src/components/YxZkCountdown/TimeCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { Countdown, TimeBox, TimeCardContainer, TimeCardLabel } from './styles/TimeCard.styled'
+type Props = {
+  days: number
+  hours: number
+  minutes: number
+  seconds: number
+}
+
+const TimeCard = (props: Props) => {
+  return (
+    <TimeCardContainer>
+      <TimeBox>
+        <TimeBox className="tophalf" />
+        <TimeBox className="bottomhalf" />
+        <TimeBox className="bar" />
+        <TimeBox className="number">{props.days}</TimeBox>
+        <TimeBox className="right-curve" />
+        <TimeBox className="left-curve" />
+        <TimeCardLabel>Days</TimeCardLabel>
+      </TimeBox>
```

Comment:
Consider making each of the TimeBox a component since the only things changing are the value to display and the label. Can consider making a component that takes in two props: value:number and label:string.

Reply:
Fixed in commit 86996b3